core/src/main/scala/org/apache/spark/shuffle/FileShuffleBlockResolver.scala

```diff
@@ -23,6 +23,8 @@ import java.util.concurrent.atomic.AtomicInteger
 
 import scala.collection.JavaConversions._
 
+import com.google.common.annotations.VisibleForTesting
```

Comment:
Ok, that's cool.  I just wasn't paying enough attention to be aware of when this new practice was introduced.  I'm in favor of it as long as it is standard practice.